### PR TITLE
Add support for POST for all endpoints

### PR DESCRIPTION
**Pull request by kbagchiGWC**
Validation : validated all endpoints using curl.

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -47,59 +62,93 @@ function tokenGenerator(identity) {
...
 /**
  * Makes a call to the specified client using the Twilio REST API.
  *
- * @param {string} to - The recipient of the call, a client.
+ * @param {Object} request - POST or GET request that provides the recipient of the call, a phone number or a client
+ * @param {Object} response - The Response Object for the http request
  * @returns {string} - The CallSid
  */
-function placeCall(to, request) {
+function placeCall(request, response) {
```
Comment (idelgado) [inline src/server.js]: This implementation differs from `java` and `python`

**Review by idelgado — CHANGES_REQUESTED** [review]

**Comment by kbagchiGWC** [comment]
Result : 

----------------------------------------------------------------------
Ran 18 tests in 15.168s

OK

File: src/server.js [hunk 2]
```diff
@@ -47,59 +61,105 @@ function tokenGenerator(identity) {
...
+  }
+
+  const voiceResponse = new VoiceResponse();
 
   if (!to) {
-      response.say("Congratulations! You have made your first call! Good bye.");
-  } else if (phoneNumberChars.indexOf(to[0]) != -1) {
-      const dial = response.dial();
+      voiceResponse.say("Congratulations! You have made your first call! Good bye.");
+  } else if (!isNaN(to)) {
```
Comment (idelgado) [inline src/server.js]: Exclude `+` sign

File: src/server.js [hunk 3]
```diff
@@ -47,59 +61,105 @@ function tokenGenerator(identity) {
...
-
-  return call.sid;
+  if (!to) {
+    console.log("Calling default client:" + defaultIdentity);
+    call = await client.api.calls.create({
+      url: url,
+      to: 'client:' + defaultIdentity,
+      from: callerId,
+    });
+  } else if (!isNaN(to)) {
```
Comment (idelgado) [inline src/server.js]: Same here

**Review by idelgado — CHANGES_REQUESTED** [review]

**Review by idelgado — APPROVED** [review]

**Review by idelgado — APPROVED** [review]

**Pull request merged**